src/main/java/user_interface/login_leaderboard/FileReader.java

```diff
@@ -18,7 +18,8 @@ public class FileReader implements IFileInput {
      */
     @Override
     public ArrayList<ArrayList<String>> readFile(){
-        String filePath = "/Users/owaiszahid/Desktop/finalstretch/src/main/java/user_interface/login_leaderboard/data.csv";
+        String filePath = "C:\\Users\\arifa\\IdeaProjects\\course-project-group-93\\src\\main\\java" +
```

Comment:
Please change this to just
```java
String filePath = "data.csv";
```
or something. Right now this will only work on your computer.

Reply:
I tried to do it that way, it did not work for me for some reason. But I changed it to "src/main/java/user_interface/login_leaderboard/data.csv"